ferrocene/doc/safety-manual/src/rustfmt/options.rst

```diff
@@ -0,0 +1,15 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Tool Options
+============
+
+``rustfmt`` is qualified exclusively for the following command line options:
+
+* Users shall pass command line option ``--emit files`` to each invocation of ``rustfmt``.
+* Users shall pass command line option ``--edition 2021`` to each invocation of ``rustfmt``.
```

Comment:
Why are we limiting to just `--edition 2021`? We do so for rustc as we only specified the 2021 edition in the FLS, but there is no such constraint for rustfmt.